src/bsoncxx/document/value.cpp

```diff
@@ -53,6 +53,9 @@ value& value::operator=(const value& rhs) {
     return *this;
 }
 
+value::value(value&& rhs) noexcept: _data{std::move(rhs._data)}, _length{std::move(rhs._length)} {
```

Comment:
I have no idea. It eliminated the error found here: https://evergreen.mongodb.com/task/mongodb_cpp_driver_dev_ubuntu1410_debug_asan_compile_and_test_67dadb7874796c9af794de652c0855c0bdc69d6c_16_01_12_22_01_34